lib/stm32/common/rtc_common_l1f024.c

```diff
@@ -120,4 +120,14 @@ void rtc_clear_wakeup_flag(void)
 	RTC_ISR &= ~RTC_ISR_WUTF;
 }
 
+/*---------------------------------------------------------------------------*/
+/** @brief Clears the Alarm A flag
+
+@details RTC_ISR[13:8] bits are not write protected: no need to unlock
+*/
+void rtc_clear_alarm_flag(void)
+{
+	RTC_ISR &= ~RTC_ISR_ALRAF;
```

Comment:
Well, it is probably better to keep it in the common code in the case if the same functionality is available on all the platforms in question. Anyway thanks for pointing at the alarm B thing, I will take a look through all those specs and get back with comments and updates. On the first glance, the answer is going to depend on whether alarm B is available on the same platforms as A.